pkg/proxy/apis/config/v1alpha1/defaults.go

```diff
@@ -64,6 +64,10 @@ func SetDefaults_KubeProxyConfiguration(obj *kubeproxyconfigv1alpha1.KubeProxyCo
 	if obj.IPTables.MinSyncPeriod.Duration == 0 {
 		obj.IPTables.MinSyncPeriod = metav1.Duration{Duration: 1 * time.Second}
 	}
+	if obj.IPTables.DisableLocalhostNodePorts == nil {
+		obj.IPTables.DisableLocalhostNodePorts = pointer.Bool(false)
```

Comment:
comment here that this is the back-compatible value, even if not the best choice.

Reply:
Done.